app/models/smart_proxy.rb

```diff
@@ -111,15 +111,15 @@ def sanitize_url
   end
 
   def associate_features
-    return true if Rails.env == 'test'
```

Comment:
I have been pulling my hair with how to fix the tests... Your idea is better. Will refactor